Enabling Ansible task for Deployment Group and merging master to release

vinodkumar3 opened the pull request.

sureshtadisetty committed:
Create SampleServiceBusMessageBody.txt

rwatjen committed:
Missing curly brackets

The template variables for data source endpoints etc. require three curly brackets, not just two. (See also e.g. the `AzureResourceGroups` data source in https://docs.microsoft.com/en-us/rest/api/vsts/serviceendpoint/types/list?view=vsts-rest-4.1#examples .

Commit:
Standard change request type added

Commit:
test changes removed

Commit:
task updated

Commit:
Task updated

Commit:
GroupName removed

Commit:
Create CR task updated

Commit:
Output variables added

amaljg committed:
Removing unused image files from Ansible extension. (#590)

nishubansal91 committed:
Output variable updated and dataSource of impact changed (#584)

Commit:
master merged and conflicts resolved

v-zbsail committed:
LOC CHECKIN | Microsoft/vsts-rm-extensions master | 20190213 (#591)

amaljg committed:
Fixing the disabled Ansible tests and enabling them (#592)

Fixing the disabled Ansible tests and enabling them.

Commit:
fixes done

Commit:
Task updated

Commit:
Review changes done and readme updated

Commit:
Usage of scripted REST api is reverted

romil07 committed:
Adding the documentation on partial download of custom artifacts (#599)

Commit:
review changes done

Commit:
V1 version task added for back compat

Commit:
Publisher name and TestConnection major required version updated

gykuma committed:
Move CircleCI extensions to Marketplace - Part 2 (#598)

* Move CircleCI artifact to Extension

Commit:
Changed the URL to query active standard templates

omeshp committed:
Update readme.md (#603)

omeshp committed:
Update readme.md (#604)

omeshp committed:
Update readme.md

omeshp committed:
Update task.json

omeshp committed:
Update mp_terms.md

omeshp committed:
Update mp_terms.md

omeshp committed:
Update mp_terms.md

omeshp committed:
Update mp_terms.md

omeshp committed:
Update mp_terms.md

omeshp committed:
Update ThirdPartyNotices.txt

omeshp committed:
Update ThirdPartyNotices.txt

omeshp committed:
Update ThirdPartyNotices.txt

omeshp committed:
Update vss-extension.json

omeshp committed:
Update readme.md

omeshp committed:
Update resources.resjson

omeshp committed:
Update README.md

omeshp committed:
Update readme.md

gykuma committed:
fix icon issues (#601)

bansalaseem committed:
Merge pull request #583 from Microsoft/nibansal/ServiceNow/StandardChangeRequest

Change request task enhancements

gykuma committed:
Use latest version of azure-pipelines-task-lib (#602)

omeshp committed:
Update dataSources.md

Commit:
Added telemetry for SQL dacpac deploy task (#600)

Commit:
Query by Correlation Id is removed and other minor bug fixes

Commit:
public is set to true

Commit:
Support for updating existing change request in gate is added

Commit:
Passing correlation_id in V2 tasks to support back compat for some cases

Commit:
Correlation Id output variable fixed

bansalaseem committed:
Merge pull request #606 from Microsoft/users/nibansal/ExistingFlowFix

Query by Correlation Id is removed and other minor bug fixes for V0 and V1 versions

gykuma committed:
Move Jenkins test data to new server (#607)

romil07 committed:
Update PartialArtifactsDownload.md (#610)

arjgupta committed:
handle encoding issue for powershell legacy handlers based iis web extension tasks (#611)

Commit:
updated circle-ci extension id and endpoint contribution ID (#612)

buckh committed:
The name of the extension has a spelling error

Commit:
refactored extension's name from circlci to circleci

Lovakumar committed:
Merge pull request #614 from buckh/patch-1

The name of the extension has a spelling error

Lovakumar committed:
Merge pull request #616 from PranaySPatil/users/pranpati/circleCIIdUpdate

Updated extension's name from circlci to circleci

Commit:
Check state of app pool (#615)

* Check state of app pool

* Fixed test cases

* Review comment

rohit-batra committed:
Updated readme to include feed location (#619)

omeshp committed:
[Security] Bump extend from 3.0.1 to 3.0.2 (#620)

Bumps [extend](https://github.com/justmoon/node-extend) from 3.0.1 to 3.0.2. **This update includes security fixes.**
- [Release notes](https://github.com/justmoon/node-extend/releases)
- [Changelog](https://github.com/justmoon/node-extend/blob/master/CHANGELOG.md)
- [Commits](https://github.com/justmoon/node-extend/compare/v3.0.1...v3.0.2)

bsriram committed:
Adding documentation for default version types for artifact contribution (#469)

omeshp committed:
[Security] Bump tar from 4.4.1 to 4.4.8 (#622)

Bumps [tar](https://github.com/npm/node-tar) from 4.4.1 to 4.4.8. **This update includes security fixes.**
- [Release notes](https://github.com/npm/node-tar/releases)
- [Commits](https://github.com/npm/node-tar/compare/v4.4.1...v4.4.8)

bansalaseem committed:
Merge pull request #330 from microsoft/users/surest/SampleServiceBusMessageBodyForRef

SampleServiceBusMessageBody.txt for reference

Commit:
Update README_IISAppMgmt.md (#624)

nschonni committed:
fix: Extensions/Ansible/Src/readme.md

- Trim trailing/leading spaces
- Remove extra bold from titles

vinodkumar3 committed:
Enabling Ansible task for Deployment Group as well

vinodkumar3 committed:
Updated Ansible extension version

Commit:
Update README_IISAppMgmt.md (#629)

vinodkumar3 committed:
Updated version of Ansible task

20shivangi committed:
Hiding password (#631)

* Hiding password

* Fixing indentation

* Changing extension

vinodkumar3 committed:
Enabling Ansible task for Deployment Group as well (#628)

* Enabling Ansible task for Deployment Group as well

* Updated Ansible extension version

* Updated version of Ansible task

vinodkumar3 committed:
Merge branch 'master' into release

vinodkumar3 commented on Extensions/Ansible/Src/Tasks/Ansible/ansibleCommandLineInterface.ts:
```diff
@@ -131,7 +130,7 @@ export class ansibleCommandLineInterface extends ansibleInterface {
         return new Promise<string>(async (resolve, reject) => {
             try {
-                let remoteInventory = '/tmp/' + uuid() + 'inventory.ini';
+                let remoteInventory = ansibleUtils.getTemporaryInventoryFilePath();
```
Its a part of 4 months old [PR](https://github.com/microsoft/azure-pipelines-extensions/pull/592/files).
@amaljg are you fine merging your PR changes to release branch now ?

sachinma replied:
why don't you just cherry-pick your PR

vinodkumar3 replied:
As I said multiple merge conflicts comes into picture in azure-pipelines-extensions\package-lock.json when I just cherry-pick my changes

jessehouwing committed:
Add Canada Central region

For the #gdbc we need to run Twitter Sentiment Analysis in Canada Central. Would love to have this added quickly!

bansalaseem committed:
Merge pull request #634 from jessehouwing/master

Add Canada Central region

vithati committed:
Changed cognitiveServicesEndpointRegion input picklist to editable picklist. (#635)

Update the extension version.

sachinma committed:
Merge pull request #627 from nschonni/fix--Extensions/Ansible/Src/readme.md

fix: Extensions/Ansible/Src/readme.md

vinodkumar3 committed:
Merge remote-tracking branch 'origin/master' into users/kuvinod-ansibledg-release
